src/app/components/Toolbar/Features/SearchAddress/SearchBox.tsx

```diff
@@ -1,6 +1,7 @@
 import { Box, Button, TextInput } from 'grommet'
 import { FormClose, Search } from 'grommet-icons/icons'
 import React from 'react'
+import styled, { useTheme } from 'styled-components'
```

Comment:
useTheme is not used

Reply:
oops :D 

I looked at some linting options to detect these, and now I've got 204 other problems